i18n/MonotoneMuseum/ko_kr.yml

```
@@ -25,65 +25,538 @@ ko_kr:
-        - "【시간가속】\n씬 내에 존재하는 모든 무생물과 생물은 2D6년만큼 시간이 가속한다. 생물이라면 노화한다. 방적공은 【봉제】 난이도 8의 판정에 성공하면 이 영향을 받지 않는다."
-        - "【시간역류】\n씬 내에 존재하는 모든 무생물과 생물은 2D6년만큼 시간이 역류한다. 생물이라면 젊어진다. 제조년/생년보다 앞으로 돌아왔을 때는 허무에 삼켜져서 소멸한다. 방적공은 【봉제】 난이도 8의 판정에 성공하면 이 영향을 받지 않는다."
-        - "【재해도래】\n폭풍우, 화산분화, 홍수 등, 흐트러짐에 의해서 어지럽혀진 자연이 사람들에게 송곳니를 드러낸다."
-        - "【인심황폐】\n일그러짐에 의해 초래된 불안과 공포는 사람들을 자포자기하게 한다."
-        - "【평온무사】\n아무것도 일어나지 않는다. 방적공들은 등골이 오싹할 정도의 공포를 느낀다."
-        - "【역병만연】\n등장하고 있는 캐릭터는 【육체】 난이도 8의 판정을 하여 실패하면 ［사독］5를 받는다. 병의 치료법의 유무 등에 대해서는 GM이 결정한다. 고민된다면 가람을 쓰러트리면 병도 소멸한다, 라고 하라."
-        - "【이단심문】\n이단심문의 시기가 가깝다. PC들이 방적공인 것이 알려진다면, PC들도 화형대로 보내지게 될 것이다."
-        - "【일그러짐출현】\n흐트러짐의 파편으로부터 일그러짐이 나타나 사람들을 습격한다. 병마（『인카르챤드』 P.238）×2와 전투를 하게 해도 좋다. 또, 다른 이형이라도 좋다."
-        - "【악몽출현】\n씬 내의 모든 사람은 무서운 공포의 꿈을 꾼다. 마음 약한 사람들은 가람에 매달리던가, 이단자들을 화형에 처하는 것으로 이 꿈에서 벗어날 수 있을 거로 생각할 것이다."
-        - "【쥐의 연회】\n다수의 쥐의 떼가 출현하여 곡물을 들쑤셔 먹고 역병을 흩뿌린다. 대형 쥐（『MM』 P.240）×1D6와 전투를 하게 해도 좋다."
-        - "【왜곡길잡이표】\n삐뚤어진 길잡이표가 내려진다. 일그러짐표（『MM』 P.263）를 굴릴 것."
-        - "【지역소멸】\n연극의 무대가 되는 지역 그 자체가 사라져서 없어진다. 영향 아래에 있는 모든 캐릭터(가람을 포함한다)는 【봉제】 난이도 10의 판정에 성공하면 탈출할 수 있다. 실패했을 경우 그 즉시 사망한다. 엑스트라는 무조건 사망한다."
+        - |-
+          【색채침식】
+          씬 내에 존재하는 온갖 무생물과 생물은 흑백으로 변한다. 실 잣는 이는 【재봉】 난이도 8의 판정에 성공하면 이 영향을 받지 않는다. 이 효과는 일그러짐을 불러온 이형이 죽으면 해제된다.
+        - |-
+          【허무출현】
+          올풀림 속에서 허무가 스며나와 힘 약한 자들을 끌고 가 소멸시킨다. 씬에 등장하고 있는 엑스트라는 벌레 한 마리 조차 남지 않고 소멸하고 두 번 다시 나타나지 않는다.
+        - |-
+          【계절변화】
+          계절이 돌변한다. 1D6을 굴려 1이면 봄, 2면 여름, 3이면 가을, 4면 겨울, 5면 플레이하는 현재의 계절, 6이면 GM의 임의로 하라. 급격한 계절의 변화는 자연히 농작물과 사람들의 건강을 상하게 한다.
+        - |-
+          【올풀림】
+          세계가 갈라져 올풀림이 출현한다. 올풀림에 닿은 자들은 허무에 삼켜져 돌아올 수 없게 된다.
+        - |-
+          【이형화】
+          가람이 낳은 절망과 공포가 배어 씬 내의 모든 엑스트라는 모종의 이형화를 받는다. 이것을 치료할 주술은 없다. 이형의 무리 (IZ P237) * 1D6과 전투시켜도 좋다. 
+        - |-
+          【죽음진행】
+          사람들의 마음에 허무가 퍼져 불안과 절망이 가득 차오른다. 도망칠 수 없는 공포로부터 도망치기 위해 사람들은 스스로, 혹은 무의식적으로 죽음으로 향하는 행동을 시작한다. 
+        - |-
+          【시간가속】
+          시간이 가속한다. 씬 내에 존재하는 여러 무생물과 생물은 2d6 년의 세월이 지난 것 처럼 행동한다. 생물이라면 노화한다. 실 잣는 이는 【재봉】 난이도 8의 판정에 성공하면 이 영향을 받지 않는다.
+        - |-
+          【시간역류】
+          시간이 국지적으로 역류한다. 씬 내에 존재하는 여러 무생물과 생물은 2d6년 전으로 돌아간다. 생물이라면 젊어진다. 이 결과, 제조년/출생년도 보다 전으로 돌아갔을 경우에는 허무에 삼켜저 소멸한다. 실 잣는 이는 【재봉】난이도 8의 판정에 성공하면 이 영향을 받지 않는다. 
+        - |-
+          【재해도래】
+          올풀림에 의해 흐트러진 자연이 사람들에게 이빨을 드러낸다. 태풍, 화산 분출, 홍수 등.
+        - |-
+          【인심 황폐】
+          일그러짐으로 인한 불안과 공포는 사람들을 자포자기하게 만든다. 노상강도가 빈발하고 관헌은 강압적이 되면서 마을은 혁명과 폭동의 소문에 휩싸이고 형벌은 잔인해진다.
+        - |-
+          【평온무사】
+          너무 섬뜩할 정도로 아무 일도 일어나지 않는다. 아무 일도 일어나지 않는다. 실 잣는 이는 등골이 서늘해지는 듯한 두려움을 느낀다.
+        - |-
+          【역병 만연】
+          고통 속에 목숨을 앗아가는 역병이 만연한다. 등장하고 있는 캐릭터는 【육체】 난이도 8의 판정을 실시해, 실패하면 사독 5를 받는다.
+        - |-
+          【이단 심문】
+          일그러짐과 올풀림의 발생을 사람들이 깨닫고, 이 재앙을 피하려면 이형을 불길에 태워야 한다고 외치기 시작한다. PC들이 실 잣는 이라는 것이 알려지면 PC들도 화형대로 보내질 것이다. 
```

The trailing space should be removed.
```suggestion
          일그러짐과 올풀림의 발생을 사람들이 깨닫고, 이 재앙을 피하려면 이형을 불길에 태워야 한다고 외치기 시작한다. PC들이 실 잣는 이라는 것이 알려지면 PC들도 화형대로 보내질 것이다.
```